pkgs/applications/science/biology/macse/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl, jre, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  pname = "macse";
+  version = "2.03";
+
+  src = fetchurl {
+    url = "https://bioweb.supagro.inra.fr/${pname}/releases/${pname}_v${version}.jar";
+    sha256 = "0jnjyz4f255glg37rawzdv4m6nfs7wfwc5dny7afvx4dz2sv4ssh";
+  };
+
+  buildInputs = [ makeWrapper ];
+
+  phases = "installPhase";
+
+  installPhase = ''
+    mkdir -p $out/share/java
+    ln -s $src $out/share/java/macse.jar
+    makeWrapper ${jre}/bin/java $out/bin/macse --add-flags "-jar $out/share/java/macse.jar"
+  '';
+
+  meta = {
```

Comment:
Would be nice to write this as `meta = with stdenv.lib; {` so then you could drop the duplicated `stdenv.lib` references below like so:
```
license = licenses.gpl2;
maintainers = [ maintainers.bzizou ];
```